packages/library-base/src/commonMain/kotlin/io/realm/MutableRealm.kt

```diff
@@ -44,19 +44,45 @@ public interface MutableRealm : TypedRealm {
     public fun cancelWrite()
 
     /**
-     * Creates a copy of an object in the Realm.
+     * Update policy for imports with [copyToRealm].
      *
-     * This will create a copy of an object and all it's children. Any already managed objects will
-     * not be copied, including the root `instance`. So invoking this with an already managed
-     * object is a no-operation.
+     * @see copyToRealm
+     */
+    // FIXME #naming
+    public enum class UpdatePolicy {
```

Comment:
I think the current names work out quite well when reading through the docs. And if staying with that there would at least be two SDKs with the same wording.

Reply:
Something I forgot to mention in my last review. Do we want this to be under `MutableRealm`? I know it is only available there, but at least to me, reading `MutableRealm.UpdatePolicy` in the code feels a bit long and annoying? 🤔 

Not sure if there is a chance it will eventually be available elsewhere, if so, we should probably make it a standalone enum.

Reply:
Thinking a bit more.... I guess `MutableRealm.UpdatePolicy` does a better job at scoping the concept, so I can probably live with the longer name.